### PR TITLE
Introduce ViewHelper, make logo configurable 

### PR DESCRIPTION
**Pull request by DarthHater**
Second verse, same as the first, more for #16 

Adds:
* ViewHelper.scala
* Moves some configuration access to that helper
* Uses ViewHelper in views where applicable

### CODE REVIEW

File: app/controllers/Application.scala [hunk 1]
```diff
@@ -64,11 +64,8 @@ class Application @Inject()
 
   val gitHubOauthScopesForClaSigning = Seq("user:email")
   val gitHubOauthScopesForAudit = Seq("read:org")
-  val maybeOrganizationName = configuration.getOptional[String]("app.organization.name")
   val maybeOrganizationUrl = configuration.getOptional[String]("app.organization.url")
```
Comment (jamesward) [inline app/controllers/Application.scala]: Should be able to move this one out too.  Then inject `ViewHelper` into the controller.

File: app/views/main.scala.html [hunk 2]
```diff
@@ -99,7 +99,7 @@
     </head>
     <body>
         <div class="slds-container--center slds-container--medium" style="text-align: center; padding: 20px;">
-            <img src="@routes.Assets.versioned("images/salesforce.svg")" style="height: 57px">
+            <img src="@viewHelper.organizationLogo()" style="height: 57px">
```
Comment (jamesward) [inline app/views/main.scala.html]: This one we can use the option instead and do that `@for() {` thing

Comment (DarthHater) [reply app/views/main.scala.html]: Sounds good!

File: app/controllers/Application.scala (hunk 1 above)
Comment (DarthHater) [reply app/controllers/Application.scala]: In: a56017b

File: app/views/main.scala.html (hunk 2 above)
Comment (DarthHater) [reply app/views/main.scala.html]: In: ec90da8

File: app/helpers/ViewHelpers.scala [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+(configuration: Configuration ) {
+  val maybeOrganizationName = configuration.getOptional[String]("app.organization.name")
+  val maybeOrganizationLogoUrl = configuration.getOptional[String]("app.organization.logo-url")
+  val maybeOrganizationUrl = configuration.getOptional[String]("app.organization.url")
+
+  def organizationName(): String = {
+    maybeOrganizationName.getOrElse("")
+  }
+
+  def organizationLogo(): Option[String] = {
```
Comment (jamesward) [inline app/helpers/ViewHelpers.scala]: I think we should drop this one and just use the val.

Comment (DarthHater) [reply app/helpers/ViewHelpers.scala]: Sounds good!

Comment (DarthHater) [reply app/helpers/ViewHelpers.scala]: In: ca8baa2

File: app/helpers/ViewHelpers.scala [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+(configuration: Configuration ) {
+  val maybeOrganizationName = configuration.getOptional[String]("app.organization.name")
+  val maybeOrganizationLogoUrl = configuration.getOptional[String]("app.organization.logo-url")
+  val maybeOrganizationUrl = configuration.getOptional[String]("app.organization.url")
+
+  def organizationName(): String = {
+    maybeOrganizationName.getOrElse("")
+  }
+
+  def organizationUrl(): Option[String] = {
```
Comment (jamesward) [inline app/helpers/ViewHelpers.scala]: Oh sorry.  Missed this one.  We can just use the val here too.

File: app/helpers/ViewHelpers.scala (hunk 3 above)
Comment (DarthHater) [reply app/helpers/ViewHelpers.scala]: Fixed a booboo in: 88e1d0c , going to add some tests as well.

File: app/helpers/ViewHelpers.scala (hunk 4 above)
Comment (DarthHater) [reply app/helpers/ViewHelpers.scala]: In: bcdccd6

File: test/helpers/ViewHelpersSpec.scala [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  "ViewHelper" must {
+    "give a valid organization name" in {
+      val orgName = viewHelper.organizationName()
+      orgName mustBe a [String]
+      orgName mustEqual(testOrgName)
+    }
+    "give a valid organization URL" in {
+      val orgUrl = viewHelper.maybeOrganizationUrl
+      orgUrl shouldBe defined
+      orgUrl shouldEqual Some(testOrgUrl)
```
Comment (jamesward) [inline test/helpers/ViewHelpersSpec.scala]: That works and you can also do `orgUrl should contain (testOrUrl)`

Comment (DarthHater) [reply test/helpers/ViewHelpersSpec.scala]: Neato. Yeah, learning my way around Scala and Optional, hah

File: test/helpers/ViewHelpersSpec.scala [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+      Map(
+        "app.organization.name" -> testOrgName,
+        "app.organization.url" -> testOrgUrl,
+        "app.organization.logo-url"-> testOrgLogoUrl
+      )
+    )
+    .in(Mode.Test)
+    .build()
+
+  lazy val viewHelper = app.injector.instanceOf[ViewHelpers]
```
Comment (jamesward) [inline test/helpers/ViewHelpersSpec.scala]: I think this needs to be a `def` at the `class` scope or things get wonky.

**Pull request merged**